app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+
+        person_instance = Person(name, age)
+
+        Person.people[name] = person_instance
+
+    for person in people:
+        current_instance = Person.people[person["name"]]
+        if person.get("wife"):
+            current_instance.wife = Person.people[person["wife"]]
```

Comment:
Ensure that the `wife` attribute is defined in the `Person` class before assigning it here. Consider initializing it in the `__init__` method or checking if it exists before assignment.